scripts/build-post-list.js

```diff
@@ -88,12 +90,23 @@ function walkDirectories(directories, result, sectionWeight = 0, sectionTitle, s
             details.title = capitalize(fileName)
           }
 
+          if(releaseNotes.includes(details.title)){
+            details.releaseNoteLink = `/blog/release-notes-${details.title}`
+          }
+
           if (fileBaseName.includes('next-spec') || fileBaseName.includes('next-major-spec')) {
             details.isPrerelease = true
             // this need to be separate because the `-` in "Pre-release" will get removed by `capitalize()` function
             details.title += " (Pre-release)"
           }
         }
+        
+        if(file.startsWith("release-notes") && dir[1] === "/blog"){
```

Comment:
please add comments in this code:
- why this if is here, what is it's purpose
- whenever there are things like `slice` or `lastIndexOf` please please add comment with sample data, so for future debugging it is clear what is the expected input

Reply:
@derberg I have added these changes, kindly check once. :grin: